src/sidebar/services/test/tags-test.js

```diff
@@ -56,6 +56,12 @@ describe('TagsService', () => {
         count: 1,
         updated: stamp,
       },
+      'žŸ¡¢£¤¥¦§': {
```

Comment:
Can you re-add the Unicode test case from the earlier version of this commit, based on the bug report. We want both since they exercise different behavior.